core/src/main/resources/defaults/blueprints/cdp-data-mart-active.bp

```diff
@@ -0,0 +1,226 @@
+{
+  "description": "CDP 1.0 Active Data Mart template with Apache Impala, Hue, and Kudu",
```

Comment:
Should this mention Spark?

Reply:
Yeah I guess so. Added it (but not YARN, which is just a dependency like HDFS).